src/generic-container/generic-container.ts

```diff
@@ -221,6 +224,11 @@ export class GenericContainer implements TestContainer {
     return this;
   }
 
+  public withLabel({ key, value }: { key: string; value: string }): this {
```

Comment:
i think that `withLabels({ [key:string]: value })` works better with JS. 

So instead of:

```typescript
withLabel({ key: 'a', value: 'a' })
withLabel({ key: 'b', value: 'b' })
```

You could simply do:

```typescript
withLabels({ a: 'a', b: 'b' })
```

Reply:
BTW, I see `withEnv` is structured similarly:
```ts
public withEnv(key: EnvKey, value: EnvValue)
```

Reply:
Yes I did think that's where you got it from 🙂  I'm planning on changing that API as well